homeassistant/components/voip/voip.py

```diff
@@ -0,0 +1,179 @@
+"""Voice over IP (VoIP) implementation."""
+import asyncio
+import logging
+import time
+
+import async_timeout
+from voip_utils import CallInfo, RtpDatagramProtocol, SdpInfo, VoipDatagramProtocol
+
+from homeassistant.components import stt, tts
+from homeassistant.components.voice_assistant import async_pipeline_from_audio_stream
+from homeassistant.components.voice_assistant.pipeline import (
```

Comment:
If we need these, we should import them from voice_assistant `__init__.py`. We shouldn’t rely on internal details of an integration.